matching-service/utils/authentication.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable consistent-return */
```

Comment:
Curious what's the rationale behind prohibiting this?

Reply:
Its mainly cos of :
```
    jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err, user) => {
      if (err) {
        console.log(err);
        return res.status(401).json({ message: "Invalid access token" });
      }
      req.user = user;
      req.accessToken = token;
      next();
    });
```

Since nothing is "returned", while in the catch block a response is returned, the rule flags it due to inconsistent returns.